_posts/2020-05-01-newsletter.md

```diff
@@ -0,0 +1,163 @@
+---------------
+
+Neovim is a fork of the venerable text-editor vim, focused on extensibility and
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html).
+
+### Participating
+
+If you are interested in contributing to Neovim, visit [the github
+page](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md) for details.
+We have a great test suite and automated CI, so you can just jump in and have a
+go. Don't be afraid to open a PR to get comments or just take advantage of the
+infrastructure!
+
+To get in touch with the team, join the [gitter channel](https://gitter.im/neovim) (also
+accessible via [IRC](https://webchat.freenode.net/?channels=neovim)), or visit
+the [Neovim subreddit](https://www.reddit.com/r/neovim/), which many of the
+developers read. Taking part in conversations is a contribution, too!
+
+What's new
+----------
+
+The [last newsletter](https://neovim.io/news/2020/04/) covered the release of Neovim v0.3.0, it is now time to do the same thing with the most exciting Neovim release that happened since then: v0.4.0!
+
+#### 0.4 for Users
+
+- Neovim started detecting the background color of the terminal for xterm-compatible terminals (**@joshtriplett**, [#9509](https://github.com/neovim/neovim/pull/9509))
+
+- The 'maxcombine' option has been removed, it now always default to 6 (**@bfredl**, [#7992](https://github.com/neovim/neovim/pull/7992))
+
+- The 'fillchars' setting now has an 'eob' option, letting users configure the character that should be used to represent lines after the end of a buffer (previously hardcoded to '~') (**@FriedSock**, [#8546](https://github.com/neovim/neovim/pull/8546))
+
+- The `F` has been added to the default `shortmess` option and `S` has been removed (**@justinmk**, [#8619](https://github.com/neovim/neovim/pull/8619), [#10136](https://github.com/neovim/neovim/pull/10136))
+
+- The CurslorLine will now be smarter: if its foreground color is not net, the text of the CursorLine will use the CursorLine's background color and default syntax highliting colors as foreground (**@zhou13**, [#8578](https://github.com/neovim/neovim/pull/8578)).
+
+- The terminal UI can now support undercurl and colored underline, provided that your terminal has support for that (**@blueyed**, [#9052](https://github.com/neovim/neovim/pull/9052)).
+
+- Neovim's `background` setting now defaults to `dark` to get a consistent setting across platforms (**@justinmk**, [#9205](https://github.com/neovim/neovim/pull/9205)).
+
+- `fillchars` and `listchars` now are window-local options (**@mhinz**, [#9539](https://github.com/neovim/neovim/pull/9539)).
+
+- The popupmenu can now be made transparent (**@bfredl**, [#9571](https://github.com/neovim/neovim/pull/9571)).
```

Comment:
GUIs may be transparent but the TUI not so I would add a `(pseudo)` transparent in front.